templates/fallback

```diff
@@ -12,7 +12,7 @@ type {{$decorator}} struct {
   interval time.Duration
 }
 
-// New{{$decorator}} takes several implementations of the {{.Interface.Type}} and returns an instance of {{.Interface.Type}} 
+// New{{$decorator}} takes several implementations of the {{.Interface.Type}} and returns an instance of {{.Interface.Type}}
```

Comment:
update the decorator's description please https://github.com/hexdigest/gowrap/pull/46/files#diff-60cb9937d4d11480926d7b48959484a1d9a4f706ffb2ea0db4f41269b56fe328R9 :)